heft-plugins/heft-jest-plugin/.npmignore

```diff
@@ -29,3 +29,4 @@
 
 # (Add your project-specific overrides here)
 !/includes/**
+!heft-plugin.json
```

Comment:
I think this is a good idea, though it may be smart to do this to all projects after Heft is merged. One big PR to get it done across the board feels more consistent than doing it piecemeal as part of a large feature change.

Reply:
For now I'll move the exclusion up to the standard list of exclusions.